source/views/streaming/radio/index.js

```diff
@@ -14,29 +14,29 @@ import {
   View,
   WebView,
 } from 'react-native'
-import * as c from '../components/colors'
```

Comment:
This file should be named something other than "index", and "index" should then only export these two components (I think Dictionary behaves in a way that I would like you to emulate.)

These two components should then be exported again from "views/streaming/index", which is the only place where other files should import from.